core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -725,6 +725,9 @@
     matchbox-client.key:
       bw_item: packet.net
       attachment: client.key
+    .slackhook:
+      bw_item: packet.net
+      attachment: .slackhook
   to:
     - cluster: api.ci
```

Comment:
Why do we duplicate everything rather than just specyfing one target per namespace? Also, why is the one namespace only needed on api.ci and not build01?

Reply:
@alvaroaleman are you asking why we have `cluster-secrets-metal` as well as `cluster-secrets-packet`?

@droslean yes let's make sure these are both on `build01` as well as `api.ci`

Reply:
addressed

Reply:
yes, we can just do another `to` entry with the same cluster but a different namespace rather than duplicating the whole thing

Reply:
I don't follow. Why do we need to create the secret in a different namespace?

Reply:
This block:
* Creates secrets in {cluster: api.ci, namespace: ci, name: cluster-secrets-metal},{cluster: build01, namespace: ci, name: cluster-secrets metal}

The block below:
* Creates the same secrets in {cluster: api.ci, namespace: ci, name: cluster-secrets-packet}, {cluster: build01, namespace: ci, name: cluster-secrets-packet}

Since everything about these two blocks except the `to` part seems identical, please merge the two `to` configs and remove one of the two `from` blocks

Reply:
the `.from` is not the same.